content/healthchecks/readinessprobe.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Configure Readiness Probe"
+chapter: true
+weight: 10
+---
+
+# Configure Readiness Probe
+
+```
+kubectl apply -f ~/environment/healthchecks/readiness-deployment.yaml
+```
```

Comment:
Can you go through the details of how readiness is configured? and highlight when you kill nodejs application what happens under the covers with Kubernetes with healthchecks?

Reply:
done